lua/tardis/cl_hud.lua

```diff
@@ -2,41 +2,73 @@
 
 surface.CreateFont("TARDIS-HUD-Large", {
     font="Roboto",
-    size=68
+    size=58
 })
 
 surface.CreateFont("TARDIS-HUD-Med", {
     font="Roboto",
-    size=40
+    size=36
 })
 
 surface.CreateFont("TARDIS-HUD-Small", {
     font="Roboto",
-    size=34
+    size=30
 })
 
-hook.Add("HUDPaint", "TARDIS-HUD", function()
-    if not TARDIS:GetSetting("health-enabled") then return end
-    if not (LocalPlayer():GetTardisData("interior") or LocalPlayer():GetTardisData("exterior")) then return end
-    local tardis = LocalPlayer():GetTardisData("exterior")
-    if not IsValid(tardis) then return end
-    local sw = ScrW()
-    local sh = ScrH()
-    local health = math.ceil(tardis:GetHealthPercent())
+local function CreatePercentageHUDPanel(text, value, offset, red_level)
+
+    red_level = red_level or 20
```

Comment:
Given that this is only called by two things and cannot be called by anything else as it's local we can just pass this in from the health one too, no need for this check

Reply:
befcee8d17bc313a91cad4e7940cbcbff17c1015